chart/values.yaml

```diff
@@ -1712,6 +1712,258 @@ mapsApi:
+    ##
+    pullSecrets: []
+  ## @param sqlWorker.replicaCount Number of sql-worker replicas to deploy
+  ##
+  replicaCount: 1
+  ## @param sqlWorker.containerPorts.http sqlWorker HTTP container port
+  ##
+  containerPorts:
+    http: 8002
+  ## Configure extra options for sqlWorker containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param sqlWorker.livenessProbe.enabled Enable livenessProbe on sqlWorker containers
+  ## @param sqlWorker.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param sqlWorker.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param sqlWorker.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param sqlWorker.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param sqlWorker.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param sqlWorker.readinessProbe.enabled Enable readinessProbe on sqlWorker containers
+  ## @param sqlWorker.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param sqlWorker.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param sqlWorker.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param sqlWorker.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param sqlWorker.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param sqlWorker.startupProbe.enabled Enable startupProbe on sqlWorker containers
+  ## @param sqlWorker.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param sqlWorker.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param sqlWorker.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param sqlWorker.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param sqlWorker.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: false
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## sql-worker resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param sqlWorker.resources.limits [object] The resources limits for the sql-worker containers
+  ## @skip sqlWorker.resources.limits
+  ## @param sqlWorker.resources.requests [object] The requested resources for the sql-worker containers
+  ## @skip sqlWorker.resources.requests
+  ##
+  resources:
```

Comment:
Please remember to update VIB's `runtime_parameters` adding both `sqlWorker.replicaCount` and `sqlWorker.resources` values. Also, please try decreasing the requested resources in VIB for this pod if possible.

Reply:
Hi @FraPazGal ! Thanks! this is a WIP, I ges we can lower the resource request without problems